quickstart-Azure.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: "Quickstart Azure DevOps"
+description: "Omg, we cant' believe you're using Microsoft"
```

Comment:
I get this is an initial skeleton - but - pls don't 🫣

Reply:
I will change it